File: src/main/java/br/com/projetojupiter/repository/CriadorRepository.java
package br.com.projetojupiter.repository;

import java.util.Optional;

import br.com.projetojupiter.model.Criador;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CriadorRepository extends JpaRepository<Criador, Long> {
	
	 Optional<Criador> findByEmail(String email);
}

File: src/main/java/br/com/projetojupiter/repository/CriadorRepository.kt
package br.com.projetojupiter.repository

import br.com.projetojupiter.model.Criador
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface CriadorRepository : JpaRepository<Criador, Long> {

    fun findByEmail(email: String): Optional<Criador>
}